cmake/Reprojucer.cmake

```diff
@@ -3853,6 +3863,15 @@ function(_FRUT_generate_entitlements_file target output_filename out_var)
     string(APPEND entitlements_content
       "\t<key>com.apple.security.app-sandbox</key>\n" "\t<true/>\n"
     )
+
+    if(JUCER_APP_SANDBOX_INHERITANCE)
+      if(JUCER_APP_SANDBOX_OPTIONS)
+        message(WARNING "Setting APP_SANDBOX_OPTIONS in addition to enabling"
+          " APP_SANDBOX_INHERITANCE can make child processes fail to launch."
+        )
+      endif()
+      string(APPEND entitlements_content "\t<key>com.apple.inherit</key>\n" "\t<true/>\n")
```

Comment:
The key should be `com.apple.security.inherit` (see https://github.com/WeAreROLI/JUCE/commit/631f1fae752f0a86ed159a2a163ab6e317106679).

Reply:
Addressed in 2e182dbf6ce2289e39d283ed39f3dcd66f29bc9d.